src/reference/asciidoc/file.adoc

```diff
@@ -411,6 +411,10 @@ Examples of such events are:
 
 This sequence of events might occur, for example, when a file is rotated.
 
+NOTE: Starting with version 5.0, a `FileTailingIdleEvent` is emitted when there is no data in the file during `idleEventInterval`.
```

Comment:
I don't think that it is a NOTE. It should be just regular narrative. Just imaging the next version and some new option. So, all of them should be in notes? No, just simple expression